src/utils/processes.ts

```diff
@@ -203,6 +217,9 @@ export const fetchAllArNSRecords = async ({
           message: e?.message,
           stack: e?.stack,
         });
+
+        emitter?.emit('arnsError', `Error getting ArNS records: ${e}`);
```

Comment:
maybe we can use the convention of "arns:<event>" here? I have seen it used elsewhere to seperate name spaces.

Reply:
[1d67dfe](https://github.com/ar-io/ar-io-sdk/pull/168/commits/1d67dfe1f7a4766bbce6cf5a55438f33423f660f)